man/fi_getinfo.3.md

```diff
@@ -514,6 +514,12 @@ fabric.  See `fi_av`(3).
 : Address is a Cray proprietary format that is used with their GNI
   protocol.
 
+*FI_ADDR_STR*
+: Address is a formatted character string.  The length and content of
+  the string is address and/or provider specific, but follows this model:
+
+  address_family://node[:service[:field3...]]
```

Comment:
Easier if this can only be passed as the src_addr, dest_addr, or node parameter.  Cannot pass this through service parameter, service param must be null if this format is used.  Can skip fields by putting "::" in string.

Core will need to help parse string.